puma/assets/configuration/spec.yaml

```
@@ -0,0 +1,15 @@
+name: puma
+files:
+- name: puma.yaml
+  options:
+  - template: init_config
+    options: []
+  - template: instances
+    options:
+    - name: control_url
+      required: true
+      description: Control URL for your puma instance
+      value:
+        example: "%%host%%:%%port_puma%%/stats?token=%%env_puma_control_token%%"
+        type: string
+    - template: instances/default
```

```suggestion
    - template: instances/http
    - template: instances/default

```

let's include the templates for http options